src/components/HotSpot.js

```diff
@@ -0,0 +1,34 @@
+// @flow
+import * as React from 'react';
+import * as PropTypes from 'prop-types';
+
+import View, { ViewPropTypes } from './View';
+
+export const HotSpotPropTypes = {
+  flow: PropTypes.shape({
+    targetId: PropTypes.string,
+    target: PropTypes.string,
+    animationType: PropTypes.string,
+  }),
+};
+
+// $FlowFixMe
+export default class HotSpot extends React.Component {
+  static propTypes = {
+    ...ViewPropTypes,
+    ...HotSpotPropTypes,
+  };
+
+  static defaultProps = {
+    name: 'HotSpot',
+  };
+
+  render() {
+    const { children, flow, ...props } = this.props;
+    return (
+      <View injectedProps={{ flow }} {...props}>
```

Comment:
so I see that for a `HotSpot` layer, you generate a normal View with the `flow` property. 

Sketch has a notion of `HotSpot` which is separated from a normal layer. I understand why you'd want to generate a normal layer but then why do we need another layer type? Could we just add another prop to `View`?

Reply:
True, yes. I actually ran into an issue where I was using my UI lib which wraps `react-primitives` and `styled-system` to turn `<View>` into `<Box>`, etc, and found myself using `<Box injectedProps={{ flow: {...} }}` which is messy. I'll move the `flow` prop to `<View>`.

Should `src/index.js` have a `HotSpot` export which is a reference to `<View>`? e.g.

```js
import View from './components/View';

const HotSpot = View;

module.exports = {
  ...
  View,
  HotSpot,
  ...
```

Might be messy though if people are importing `HotSpot` and it is removed in the future.

Reply:
let's forget about `HotSpot` for now then